task/tekton-catalog-publish/0.1/README.md

```diff
@@ -0,0 +1,82 @@
+# Tekton Catalog Publish
+
+This task publishes each Task in a Tekton catalog as [Tekton Bundles](https://tekton.dev/docs/pipelines/tekton-bundle-contracts/).
+The catalog must be structured as a [Tekton Catalog](https://github.com/tektoncd/catalog#catalog-structure).
+
+Bundles are published to `$TARGET/<task-name>:<task-version>` and, when `TAG` is provided, to
+`$TARGET/<task-name>:$TAG`. An example of extra tag could be the git sha of the catalog repo that
```

Comment:
I can guess how target is constructed from the parameters, but I think replacing it with $REGISTRY and $PATH would make this text clearer.

Reply:
done